openshift-ci/Dockerfile.tools

```diff
@@ -54,6 +54,11 @@ RUN mkdir -p $GOPATH/src/github.com/operator-framework \
     && make dep \
     && make install
 
+COPY --from=quay.io/openshift/origin-operator-registry:latest /bin/initializer /bin/initializer
```

Comment:
Because `FROM quay.io/openshift/origin-operator-registry:latest` would be replaced, we need these binaries.